lookout/style/typos/templates/quality_on_identifiers.md.jinja2

```diff
@@ -23,8 +23,17 @@ Checked tokens inside the vocabulary {{ tokens | intersect(vocabulary_tokens) |
 
 ## Identifiers correction quality
 
-{% set s = {"cumsum": 0} %}
+{% set typoed = identifiers["wrong"].ne(identifiers["correct"]).values %}
+{% set corrected = identifiers["sugg 0"].ne(identifiers["wrong"]).values %}
+
+{% set accuracies = {-1: 0} %}
 {% for pos in range(n_candidates) %}
-{% do s.__setitem__("cumsum", s["cumsum"] + 100 * (identifiers["sugg " + pos|string].eq(identifiers["correct"]).mean())) %}
-{{ "%.2f%% of <=%d-th suggestions are correct " % (s["cumsum"], pos + 1) }}
+{% do accuracies.__setitem__(pos, accuracies[pos - 1] + 100 * (logical_and(identifiers["sugg " + pos|string].eq(identifiers["correct"]), typoed).sum() / typoed.sum())) %}
```

Comment:
```suggestion
{% do accuracies.__setitem__(pos, accuracies[pos - 1] + 100 * (identifiers["sugg " + pos|string].eq(identifiers["correct"]) & typoed).sum() / typoed.sum())) %}
```

Reply:
It doesn't work like that, we even discussed it here already.

Reply:
Do you get a template parsing error?

Reply:
Yes, unknown symbol or smth like that

Reply:
Whatever, then `arr1.__and__(arr2)`